mi/dataset/driver/zplsc_c/zplsc_c_echogram.py

```diff
@@ -0,0 +1,300 @@
+"""
+@package mi.dataset.driver.zplsc_c
+@file mi/dataset/driver/zplsc_c/zplsc_c_echogram.py
+@author Craig Risien/Rene Gelinas
+@brief ZPLSC Echogram generation for the ooicore
+
+Release notes:
+
+This class supports the generation of ZPLSC-C echograms. It needs matplotlib version 1.3.1 for the code to display
+the colorbar at the bottom of the figure. If matplotlib version 1.1.1 is used, the colorbar is plotted over the
+figure instead of at the bottom of it.
+"""
+
+from datetime import datetime
+import numpy as np
+import matplotlib
+from matplotlib.dates import date2num, num2date
+from modest_image import imshow
+import mi.dataset.driver.zplsc_c.zplsc_functions as zf
+
+matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+
+__author__ = 'Craig Risien/Rene Gelinas'
```

Comment:
See http://web.archive.org/web/20111010053227/http://jaynes.colorado.edu/PythonGuidelines.html#module_formatting

`__author__ = 'Craig Risien, Rene Gelinas'`

Reply:
Updated the author and the ratio is about 50/50.  The ZPLSCCPlot is reused code.